### ISSUE TITLE
[needs:pm] Add an option similar to 'Suspend network activity until all filter lists are loaded' from uBlock Origin

### ISSUE THREAD

**Issue by githubuser013** [AdguardTeam/AdguardBrowserExtension#2809]
### Issue Details

Hi, is it possible to to add this option in adguard at least for firefox. its recommended to use, and it will be very useful to us and make sure nothing bypass adguard especially in update time and with (Open previous windows and tabs) option on firefox.
Now its enabled by default in ublock origin for firefox desktop and android.
![Suspend network activity until all filter lists are loaded](https://github.com/AdguardTeam/AdguardBrowserExtension/assets/71437659/9429e08d-aeb5-4295-a929-b7e85a4ae1b7)

**Comment by alexx7311** [comment]
@githubuser013 Hello! Thanks, we'll think about it!

**Comment by oscarorange** [comment]
I'm wondering if it's possible to create a user script to achieve this.

My browser closes often on Android, so blocking frequently doesn't apply when navigating to links from external apps.

It goes without saying that it would be ideal if this functionality was added to AdGuard, but until then it would be nice to have a solution that isn't using an alternative blocker with AdGuard filters, as AdGuard filters are best used with AdGuard and AdGuard Extra.

**Comment by krystian3w** [comment]
https://github.com/AdguardTeam/AdguardBrowserExtension/issues/2809#issuecomment-2828507664 You can try to write a script/app what will convert the links to such with a delayed opening on the fly (this may require some free web server without ads or a local one on your phone).

**Comment by githubuser013** [comment]
Dear @windwak3r its been a year since this request is there any update?
as you can see there many users request the same thing and some are effected by this in different way like
#3176

**Comment by windwak3r** [comment]
@krystian3w @githubuser013 For now we are prioritizing other features and we haven't planned to implement this feature in any particular release.

**Comment by ancientvamp** [comment]
Yeah none of my user filters load first before the websites. Guess I'll switch back to Ublock Origin.

**Comment by eiqnepm** [comment]
> Yeah none of my user filters load first before the websites. Guess I'll switch back to Ublock Origin.

It really is an essential feature

Especially on Android where Firefox isn't loaded 24/7 for obvious reasons

Opening a link from another app completely bypasses the filters which is a privacy and security issue
